content/en/news/releases/1.9.x/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 1.9.x Releases
+description: Announcements for the 1.9 release and its associated patch releases.
+weight: 21
```

Comment:
https://deploy-preview-8910--preliminary-istio.netlify.app/latest/news/releases/ shows 1.9 after 1.8, so I guess this is wrong

Reply:
Fixed